.github/workflows/release.yml

```diff
@@ -29,4 +29,22 @@ jobs:
     with:
       skip-test: true
       role-to-assume: ${{ inputs.role-to-assume }}
-      project-name: ${{ inputs.project-name }}      
+      project-name: ${{ inputs.project-name }}
+  
+  trigger-downstream:
+    needs: [release]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        repository: [chat-components]
+    steps:
+      - name: Trigger release in downstream repository
+        run: |
+          curl -L \
```

Comment:
There is github-cli available in this environment: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-github-cli-in-workflows

Which you could use instead of raw curl commands

Reply:
this particular action is only available as an extension: https://github.com/mdb/gh-dispatch

Reply:
:(